app/models/application_record.rb

```diff
@@ -1,3 +1,17 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  class << self
+    # A check to see if MySQL is being used
+    #
+    def is_mysql?
+      DatabaseConfig.adapter.downcase == 'mysql2'
+    end
+
+    # A check to see if Postgres is being used
+    #
+    def is_postgres?
+      DatabaseConfig.adapter.downcase == 'postgresql'
```

Comment:
I thought we said these should not be exceptions to the rule with `DatabaseConfig` being used and we should just use Rails? I'd expect to see something like `ActiveRecord::Base.connection.class.name == 'ActiveRecord::ConnectionAdapters::MysqlAdapter'` and similar code instead.

Reply:
The class can't be used properly :( Spring/Rails/Magic caches it between tests and makes the test suite really 💩